source/sass/components/hr-emphasis.scss

```diff
@@ -1,6 +0,0 @@
-.hr-emphasis {
```

Comment:
Moved to https://github.com/mozilla/foundation.mozilla.org/pull/6556/files#diff-4ec33c16c98a1187e560df8e66a800d8f2c2f38680138e36b5cf80962b7f36a2